util/CommonConstants.go

```diff
@@ -44,6 +46,9 @@ const (
 	TeeCommand                   = "tee"
 	LogFileName                  = "main.log"
 	NewLineChar                  = "\n"
+	ArtifactSourceType           = "CI-RUNNER"
+	ArtifactMaterialType         = "git"
+	TerminationLogDir            = "/dev/termination-log"
```

Comment:
I think this is the default log location path, we can move this constant to common-lib and add the same path in ci-runner pod template from devtron swell